packages/@aws-cdk/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -863,6 +879,158 @@ describe('serverless cluster', () => {
 
 
   });
+
+  test('can create a Serverless cluster without vpc', () => {
+    // GIVEN
+    const stack = testStack();
+
+    // WHEN
+    new ServerlessCluster(stack, 'Database', {
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
+      parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql10'),
+    });
+
+    // THEN
+    expect(stack).toHaveResource('AWS::RDS::DBCluster', {
```

Comment:
This assertion is too broad for this test. Let's use `haveResourceLike()`, and assert only on `Engine` and `EngineMode`.

Reply:
👍🏻